packages/node_modules/@webex/plugin-meetings/src/common/logs/request.js

```diff
@@ -0,0 +1,44 @@
+import uuid from 'uuid';
+
+import LoggerProxy from './logger-proxy';
+
+/**
+ * @class LogsRequest
+ */
+export default class LogsRequest {
+  /**
+   *
+   * @param {Object} options
+   * @param {Object} options.webex Webex SDK instance
+   * @param {ClientMetrics} metrics
+   */
+  constructor(options) {
+    this.webex = options.webex;
+  }
+
+  /**
+   * Uploads logs to the support server
+   *
+   * @param {Object} [options={}]
+   * @param {String} [options.feedbackId=uuid.v4] ID used for tracking
+   * @param {String} [options.locusId]
+   * @param {Stirng} [options.correlationId]
+   * @returns {Promise.<String>} Feedback ID
+   * @memberof LogsRequest
+   */
+  async uploadLogs(options = {}) {
+    const id = options.feedbackId || uuid.v4();
+    const {locusId, correlationId} = options;
+
+    LoggerProxy.logger.info(`LogsRequest->uploadLogs# uploading user logs for feedbackId: ${id}`);
+
+    try {
+      await this.webex.internal.support.submitLogs({feedbackId: id, locusId, correlationId});
```

Comment:
Why did we rename `id` if we were going to rename it back to `feedbackId` for this call?